README.md

```diff
@@ -212,7 +212,7 @@
   # Good
   pid = self()
   def new(), do: %MapSet{}
-  config = IEx.Config.new
+  config = IEx.Config.new()
```

Comment:
The rule as stated is about local function calls, so the omission here seems intentional.